agent/config/config.go

```diff
@@ -54,13 +54,33 @@ const (
 	// DefaultDockerStopTimeout specifies the value for container stop timeout duration
 	DefaultDockerStopTimeout = 30 * time.Second
 
+	// DefaultImageCleanupTimeInterval specifies the default value for image cleanup duration. It is used to
+	// remove the images pulled by agent.
+	DefaultImageCleanupTimeInterval = 30 * time.Minute
+
+	// DefaultNumImagesToDeletePerCycle specifies the default number of images to delete when agent performs
+	// image cleanup.
+	DefaultNumImagesToDeletePerCycle = 5
+
+	//DefaultImageDeletionAge specifies the default time duration that an image
+	// can be deleted since it has been pulled.
+	DefaultImageDeletionAge = 1 * time.Hour
+
 	// minimumTaskCleanupWaitDuration specifies the minimum duration to wait before cleaning up
 	// a task's container. This is used to enforce sane values for the config.TaskCleanupWaitDuration field.
 	minimumTaskCleanupWaitDuration = 1 * time.Minute
 
 	// minimumDockerStopTimeout specifies the minimum value for docker StopContainer API
 	minimumDockerStopTimeout = 1 * time.Second
 
+	// minimumImageCleanupInterval specifies the minimum time for agent to wait before performing
+	// image cleanup.
+	minimumImageCleanupInterval = 10 * time.Minute
+
+	// minimumNumImagesToDeletePerCycle specifies the minimum number of images that to be deleted when
+	// performing image cleanup.
+	minimumNumImagesToDeletePerCycle = 1
```

Comment:
This is a minimum, the intention here are every time when perform the image cleanup, at least 1 image should be delete(if there are images that are available to remove), otherwise the agent should turn off the image cleanup.